src/Webapi/Dom/Webapi__Dom__HtmlFormControlsCollection.res

```diff
@@ -0,0 +1,46 @@
+/**
+ * Spec: https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#htmlformcontrolscollection
+ */
+type t = Dom.htmlFormControlsCollection
+
+include Webapi__Dom__HtmlCollection.Impl({
+  type t = t
+})
+
+type t_namedItem = [
```

Comment:
the approach in this file could use some extra attention IMO, I'm not very set on it so would be happy to make changes

Reply:
I'm not sure why, would've been a long time ago now. I don't have a preference either way